ChangeLog.d/mbedtls_ssl_cert_cb.txt

```diff
@@ -0,0 +1,4 @@
+Features
+   * Add server certificate selection callback near end of Client Hello
```

Comment:
The entry should mention `mbedtls_ssl_conf_cert_cb()` so that interested users know where to look at next.

Reply:
:rocket: